_includes/configuration.md

```diff
@@ -55,7 +57,8 @@ database.
 The `postgresql.conf` file used with {{ site.data.product.title_short }} databases requires
 specific settings for correct operation. For example, it must correctly
 reclaim table space, control session timeouts, and format the PostgreSQL
-server log for improved system support. It is recommended that external databases use a
+server log for improved system support. Due to these requirements, Red
+Hat recommends that external {{ site.data.product.title_short }} databases use a
```

Comment:
I think you wanted the old one to avoid the "Red Hat" wording.

Reply:
Red Hat is removed.